src/data/language.js

```diff
@@ -0,0 +1 @@
+self.port.emit('language', navigator.language);
```

Comment:
Do we only want the primary language? or the full array of `navigator.languages`?